RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Automated releases
+TBD
+
+# Manual releases
+1. Set BUILD_VERSION, e.g. 1.0.0-beta.3. Ensure this tag exists for contiv-ui repo.
```

Comment:
The build process in step 2 will fail if the `contiv-ui` repo doesn't have the tag for the specified BUILD_VERSION, so not sure if this needs to be explicitly pointed out